packages/management/addons-manager/1.4.0/upstream-package.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
```

Comment:
package config should be in package.yaml

Reply:
renamed file name to package.yaml